### PR TITLE
Add some python modules for running Kubernetes tests

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: calico_test/requirements.txt [hunk 1]
```diff
@@ -10,4 +10,6 @@ netaddr
 prettytable
 git+https://github.com/projectcalico/libcalico.git
 flask
-gunicorn
\ No newline at end of file
+gunicorn
+monotime
```
Comment (caseydavenport) [inline calico_test/requirements.txt]: What do we use monotime for?

Comment (tomdee) [inline calico_test/requirements.txt]: Your perf stuff.

**Comment by caseydavenport** [comment]
LGTM - when and how does this container get built / pushed to DockerHub?

**Comment by tomdee** [comment]
@caseydavenport There's an automated docker hub build job.

**Pull request merged**